app/presenters/speech_presenter.rb

```diff
@@ -0,0 +1,41 @@
+class SpeechPresenter < ContentItemPresenter
+  def delivery_type
+    if content_item.document_type == "authored_article"
+      I18n.t("formats.speech.written_on")
+    else
+      I18n.t("formats.speech.delivered_on")
+    end
+  end
+
+  def delivered_on_metadata
+    "#{delivered_on}#{speech_type_explanation}"
+  end
+
```

Comment:
Could you update the commit message if the [from](https://github.com/alphagov/government-frontend/blob/122ae292fa540059ee165a94f8129d873c5205d5/app/presenters/speech_presenter.rb#L22-L26) and [speaker_without_profile](https://github.com/alphagov/government-frontend/blob/122ae292fa540059ee165a94f8129d873c5205d5/app/presenters/speech_presenter.rb#L50-L52) methods are no longer needed or will live somewhere else, otherwise at first glance it looks like the methods were accidentally missed.